test/routes/routes.test.js

```diff
@@ -2,26 +2,26 @@ const chai = require('chai');
 const request = require('request-promise-native');
 const app = require('../../src/app');
 const getAllRoutes = require('../services/helpers/getAllRoutes');
-const { whitelist, ignoreList } = require('./whitelist');
+const { whitelistNoJwt, whitelistInvalidJwt, ignorelistNoJwt, ignorelistInvalidJwt } = require('./whitelist');
```

Comment:
Codacy found an issue: [Expected a line break after this opening brace.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5673434)